packages/vue/src/components/list/MultiDropdownList.jsx

```diff
@@ -250,7 +259,6 @@ const MultiDropdownList = {
 
 		handleChange(item) {
```

Comment:
@ShahAnuj2610 You need to make `handleChange` changes here too.

Reply:
we don't need here, it directly accepts `item` as a param rather than `event`